graphics/transform_struct.h

```diff
@@ -35,6 +35,22 @@ enum TSpriteBlendMode {
 	NUM_BLEND_MODES
 };
 
+/**
+ @brief The possible flipping parameters for the blit method.
+ */
+enum FLIP_FLAGS {
+	/// The image will not be flipped.
+	FLIP_NONE = 0,
+	/// The image will be flipped at the horizontal axis.
+	FLIP_H = 1,
+	/// The image will be flipped at the vertical axis.
+	FLIP_V = 2,
+	/// The image will be flipped at the horizontal and vertical axis.
+	FLIP_HV = FLIP_H | FLIP_V,
+	/// The image will be flipped at the horizontal and vertical axis.
+	FLIP_VH = FLIP_H | FLIP_V
```

Comment:
Isn't that the same as FLIP_HV above?

Reply:
It is, but it already existed before when the enum was in `graphics/transparent_surface.h`. Should I remove it anyway?

Reply:
It is just a convenience thing. Let it stay